west.yml

```diff
@@ -188,7 +188,7 @@ manifest:
       groups:
         - hal
     - name: hal_nuvoton
-      revision: cf24f9ebd893b349cec439ef7eaac6554a6b6d6f
+      revision: pull/10/head
```

Comment:
`34efb92e37bd07043a2cab7fff847d8443d930f9`

Reply:
Thanks for reminding, it's done.